addons/edi_notifier/tests/test_edi_notifier.py

```diff
@@ -51,25 +56,70 @@ def create_transfer(self, doc):
             }
         )
         # Create transfers
-        self.transfer = EdiTransfer.create({"gateway_id": self.gateway.id,})
+        self.transfer = EdiTransfer.create(
+            {
+                "gateway_id": self.gateway.id,
+            }
+        )
         self.transfer.doc_ids += doc
 
     def setup_cron(self, nextcall_time):
         IrCron = self.env["ir.cron"]
         action = self.notifier.action_view_cron()
         self.cron = IrCron.with_context(action["context"]).create(
-            {"name": "Test cron job", "nextcall": self._convert_time(nextcall_time),}
+            {
+                "name": "Test cron job",
+                "nextcall": self._convert_time(nextcall_time),
+            }
         )
 
     def _convert_time(self, time):
         return fields.Datetime.to_string(time)
 
     def make_note(self, res, text):
         self.env["mail.message"].create(
-            {"model": res._name, "res_id": res.id, "body": text,}
+            {
+                "model": res._name,
+                "res_id": res.id,
+                "body": text,
+            }
         )
 
 
+class TestDisabledEmailNotifier(EdiNotifierCase):
+    """Disable email notifier test"""
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        EdiNotifier = cls.env["edi.notifier"]
+        IrModel = cls.env["ir.model"]
+        cls.doc_type_raw = cls.env.ref("edi.raw_document_type")
+        cls.email_template = cls.env.ref(
+            "edi_notifier.email_template_edi_document_not_received"
+        )
+        cls.notifier = EdiNotifier.create(
+            {
+                "name": "Email on test doc missing",
+                "model_id": IrModel._get_id("edi.notifier.email.missing.in.range"),
+                "doc_type_ids": [(6, False, cls.doc_type.ids)],
+                "template_id": cls.email_template.id,
+                "lookback_hours": 3,
```

Comment:
Just a suggestion - Can we make this configurable rather than static value?

Reply:
This is a test, do we need config?

Reply:
this wasn't part of this PR anyway, just formatting I think!